### DISCUSSION TITLE
Is there any reason why I would not want to export my passphrase as well?

### DISCUSSION THREAD

**Discussion by sudwhiwdh** [borgbase/vorta#1648] [FAQ]
![Screenshot](https://user-images.githubusercontent.com/58292468/224721136-0c73b686-f8ac-41d6-9524-8d74f5a16d0c.png)

If I have protected my repository with a passphrase and want to use Vorta again on a freshly installed system, for example, I will need it again, won't I?

**Comment by real-yfprojects** [comment]
You will need it. However one might not want to store the key as plain text which is done when exporting it with the profile.

**Reply by sudwhiwdh** [reply]
Oh, that's not self-explanatory for me when I look at the export dialogue.

Why does Vorta offer me the option to store my key as plain text if this is not a secure option for sensitive data such as a backup?

Wouldn't it be possible to export this key only secured as a default setting, or to inform users that the key is not part of the export for security reasons and, if necessary, give them advice on possible secure alternatives for storing their key?

And what exactly are my options as a Vorta user if I don't want to have the key I need stored unsecured on my hard drive?

**Reply by real-yfprojects** [reply]
This is quite self exlanatory in the sense that there is no other way since passwords can only be transferred in plaintext. If you'd encrypt it you would need a password for set encryption that has to be transferred somehow. The only way to transfer passwords is your memory.

To be clear we are talking about exports here. Vorta usually uses your system keyring to store passwords. If you use a secure secret service provider like KeePassXC your data is save. But this is not the place to go into the details of credential security and cryptography.
